### PR TITLE
UI: Added expand/collapse button for single nodes and increase default value for zoom out  in lineage

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

- Added expand/collapse button for a single node
- Increase default value for zoom out

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/71748675/215260008-297356bd-a626-4b03-8381-d0f09bf8e4a6.mov

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx [hunk 1]
```diff
@@ -479,6 +479,89 @@ const EntityLineageComponent: FunctionComponent<EntityLineageProp> = ({
...
+      try {
+        const res = await getTableDetails(expandNode.id, ['columns']);
+        const tableId = expandNode.id;
+        const { columns } = res;
+        tableColumnsRef.current[tableId] = columns;
+        updateColumnsToNode(columns, tableId);
+      } catch (error) {
+        showErrorToast(
+          error as AxiosError,
+          `Error while fetching ${getDataLabel(
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx:497]: Can we add localization here?

**Review by aniketkatkar97 — COMMENTED** [review]
Left one comment. Please check

File: openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx:497]: We can add in a separate PR for now I just move the old function from bottom to top!

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx]: done

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**